content/eu/main.md

```diff
@@ -12,8 +16,48 @@ subtitle: The homepage of the European Galaxy community
 
 The **Galaxy Community** is actively involved in helping the ecosystem improve and sharing scientific discoveries.
 
+<br><br><br>
+
 <div class="row justify-content-center">
```

Comment:
(None of the links of this section are correct, I'll need to create the other pages first)
Question: Can I create cards instead of buttons? I guess that means having Vue components within the markdown file...

Reply:
Do you mean a Bootstrap card like [this](https://getbootstrap.com/docs/4.0/components/card/#example)? All you need for that is just HTML with some special classes, like in the example code in that link.

Reply:
Yes, like what we use in the general home called `HomeProfile`

Reply:
Cool, yeah then all you need is HTML. Take a look at the HTML in the Bootstrap example, or in [HomeProfile.vue](https://github.com/galaxyproject/galaxy-hub/blob/master/src/components/HomeProfile.vue) if you'd prefer that style.